.github/workflows/unit_test.yaml

```diff
@@ -43,9 +50,34 @@ jobs:
       - name: install vqpy
         shell: bash
         run: |
-          pip install 'vqpy @ git+https://github.com/vqpy/vqpy.git'
+          pip install . -v
```

Comment:
To use most up-to-date code, vqpy should be installed from "local" (files checked-out with GitHub Action) instead from the main branch.

Reply:
Since end-to-end tests can not be tested on pull request, and can only be tested after push, maybe we should test the end-to-end tests with `'vqpy @ git+https://github.com/vqpy/vqpy.git'` on push, and test the unit tests and the lint with `pip install . -v` on pull request. What do you think?

Reply:
Maybe add a new yaml file like `e2e_test.yaml` to achieve that?

Reply:
- lint.yaml: runs flake8 in any repo, on push to main branch / pull request
- unit_test.yaml: runs when push to main branch
- unit_test_external.yaml: runs in forks, on push. Owner of fork needs
  to set up secrets / file downloads themselves to run end to end tests